content/docs/pages/docs/getting-started.mdx

```diff
@@ -96,6 +96,8 @@ if you require additional configurations such as running in debug mode or switch
    ./target/release/screenpipe
    ```
 
+5. for building the desktop app then continue to follow the README for [`screenpipe-app-tauri`](https://github.com/mediar-ai/screenpipe/tree/main/screenpipe-app-tauri/README.md)
```

Comment:
@louis030195 @m13v : Is there a method to see the desktop app via homebrew install method? If not, should likely update docs to make this clear.

Reply:
atm we sell the desktop app so it's not in brew, thanks for suggestion